templates/admin/usage_dashboard.html

```diff
@@ -39,4 +49,38 @@ <h2>{{ _('Platform Totals (Last ' + months + ' Months)' )}}</h2>
         </table>
     </div>
 </div>
+<script>
+
+let timeframe = document.getElementById('timeframe')
+
+$(document).ready(function() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const months = urlParams.get('months').toString()
+
+    switch(months) {
```

Comment:
How about re-writing the switch-case(line 60-78) to the following one-liner?
```
timeframe.value = { '1': '1', '3': '3', '6': '6', '12': '12', 'all': 'all' }[months] || null;
```

Reply:
beautiful, fixed via https://github.com/bloomberg/pybossa-default-theme/pull/405/commits/bf956bb8cbad77454e8add71856f4f576716b125